Source/CarthageKit/Dependency.swift

```diff
@@ -125,8 +129,10 @@ extension Dependency: Scannable {
 				if let url = URL(string: urlString) {
 					if url.scheme == "https" || url.scheme == "file" {
 						return .success(self.binary(url))
+					} else if url.scheme == nil {
+						return .success(self.binary(URL(fileURLWithPath: url.relativePath, isDirectory: false, relativeTo: base)))
```

Comment:
Hi @blender , just realized this API required MacOS 10.11, where as this projects target is 10.10 (Current prod available 10.13), are you aware of other easy alternatives for backwards compatibility? I'm not aware of this projects compatibility goals?

Reply:
Potentially I could leverage NSURL interface, then case back to as URL
let url = NSURL(fileURLWithPath: "help", isDirectory: false, relativeTo: nil) // Optional init vs required vs latest URL API currently in code